### PR TITLE
feat: Limit name and description characters

### LINKED ISSUE #135: Limit the amount of characters accepted in the name and description columns
The name field must not exceed 32 characters, while the description field can have a maximum of 100 characters. Once we've implemented the migration to apply these constraints, it's important to handle any exceptions that may arise and return a 400 error to the user.

### PR DESCRIPTION
**Pull request by kevinszuchet**
Closes #135 

Related to #119

### CODE REVIEW

**Review by LautaroPetaccio — COMMENTED** [review]
It looks great! I've just left a small comment to check.

File: src/ports/lists/schemas.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      nullable: true
+    }
+  },
+  anyOf: [
+    {
+      required: ['name']
+    },
+    {
+      required: ['private']
+    }
```
Comment (LautaroPetaccio) [inline src/ports/lists/schemas.ts:57]: Wouldn't we need the `description` here to allow users to edit only the description of their lists?

Comment (kevinszuchet) [reply src/ports/lists/schemas.ts:57]: Yep, you're totally right. It was wrong also with my previous implementation.

**Review by LautaroPetaccio — APPROVED** [review]
Ship it 🛥️

**Pull request merged**